packages/side-nav/src/SideNav/SideNav.tsx

```diff
@@ -5,12 +5,15 @@ import PropTypes from 'prop-types';
 import { validateAriaLabelProps } from '@leafygreen-ui/a11y';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { useEventListener, useIdAllocator } from '@leafygreen-ui/hooks';
-import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
+import LeafyGreenProvider, {
+  useDarkMode,
+  useUsingKeyboardContext,
+} from '@leafygreen-ui/leafygreen-provider';
 import { keyMap } from '@leafygreen-ui/lib';
 import { useUpdatedBaseFontSize } from '@leafygreen-ui/typography';
 
-import CollapseToggle from './CollapseToggle';
-import SideNavContext from './SideNavContext';
+import { CollapseToggle } from '../CollapseToggle/CollapseToggle';
```

Comment:
We should have `index.tsx` in the subdirs to make this import statement shorter!

Reply:
That makes sense, I'll update these